### ISSUE TITLE
Vision capabilities to Chat?

### ISSUE THREAD

**Issue by ian-cameron** [jodendaal/OpenAI.Net#87]
Do you have plans to add support for [Vision](https://platform.openai.com/docs/guides/vision) to the Chat Completions API?  I wanted to check before taking any deeper look.  Thanks!

**Comment by jodendaal** [comment]
Yes would be great to add in, I could possibly look at adding this in week or so, or PR 's are welcome if you fancy adding it in.
